javascript/javascript1/week2/fullName.js

```diff
@@ -0,0 +1,24 @@
+//Flight booking getFullname function with use of formals:
+function getFullname(firstname, surname, useFormalName, isMistress) {
+  if (firstname === "" && surname === "") {
+    return "Please enter your firstname and surname";
+  } else if (useFormalName) {
+    if (isMistress) {
+      return `Mistress ${firstname} ${surname}`;
```

Comment:
You've repeated `${firstname} ${surname}` 3 times. There is a coding principle called [DRY (Don't Repeat Yourself)](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) which recommends storing code like this in variables or functions to avoid repetition.

Reply:
Done...